views/pages/login.ejs

```diff
@@ -0,0 +1,20 @@
+<%- contentFor('body') %>
+<link rel="stylesheet" type="text/css" href="/css/login.css">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    
+    <div class="imgCenter">
+        <img src="https://bhaz.com.br/wp-content/uploads/2016/11/catioro-reflexivo.jpg" alt="cachorro sorrindo" class="imgLogin">
```

Comment:
Esse catioro não é muito reflexivo, ele parece levemente irritado na real...

Reply:
Ia ser massa se o catioro fosse reflexivo __e__ responsivo! 

ba dum... ts

Ps.: Não é uma cobrança, é só pela piada ruim 😛

Reply:
![not our divisiont](https://media.giphy.com/media/5b0S9iVQ4ZO0w/giphy.gif)
Responsividade é outra história ;)